client/src/pages/CreateRecord/View/index.tsx

```diff
@@ -44,34 +46,42 @@ export default function View() {
   };
 
   const onFinish = async (values: any) => {
-    const fetchedData = await addAnimal(axios, values);
+    let fetchedData;
+    if (typeOfItem === 'Animals') {
+      fetchedData = await addAnimal(axios, values);
+    } else {
+      fetchedData = await addPlant(axios, values);
+    }
     console.log(fetchedData);
```

Comment:
add message prompt for success then navigate back to table

Reply:
Noted.

Reply:
Resolved with commit 109dc50